cirq-web/cirq_ts/src/bloch_sphere/components/vector.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 The Cirq Developers
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {ArrowHelper, Vector3} from 'three';
+
+/**
+ * Adds a state vector to the bloch sphere.
+ * @param vectorData information representing the location of the vector.
```

Comment:
@param and @returns is for a function; no need for those annotations here.

Reply:
Sloppy on my part..forgot to move